File: app/src/main/java/com/example/rickmortymvvm/models/CharacterResponseVO.java
package com.example.rickmortymvvm.models;

import java.util.List;

public class CharacterResponseVO {
    private CharacterResponseInfoVO info;
    private List<Character> results;

    public CharacterResponseInfoVO getInfo() {
        return info;
    }

    public List<Character> getResults() {
        return results;
    }
}

File: app/src/main/java/com/example/rickmortymvvm/data/remote/models/CharacterResponseVO.kt
package com.example.rickmortymvvm.data.remote.models

import com.google.gson.annotations.SerializedName

data class CharacterResponseVO(
    @SerializedName("info")
    val info: CharacterResponseInfoVO,
    @SerializedName("results")
    val results: List<CharacterRemote>
)
